### PR TITLE
add temperature dependense in SEI model

### PR DESCRIPTION
**Pull request by Vaibhav-Chopra-GT**
# Description
For issue #1695 
Multiplied the exponent in EC reaction limited, reaction limited, and interstitial diffusion-limited in SEI models with 
`1 / (1 + self.param.Theta * T)`, where `T = variables["Negative electrode temperature"]`

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [x] Code is commented, particularly in hard-to-understand areas
- [x] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Review by tinosulzer — CHANGES_REQUESTED** [review]
Thanks @Vaibhav-Chopra-GT , see comments

File: pybamm/models/submodels/interface/sei/sei_growth.py [hunk 1]
```diff
@@ -84,9 +84,15 @@ def get_coupled_variables(self, variables):
         if self.options["SEI"] == "reaction limited":
             # alpha = param.alpha
             C_sei = param.C_sei_reaction
+            T = variables["Negative electrode temperature"]
 
             # need to revise for thermal case
-            j_sei = -(1 / C_sei) * pybamm.exp(-0.5 * (delta_phi - j * L_sei * R_sei))
+            j_sei = (
+                -(1 / C_sei)
+                * pybamm.exp(-0.5 * (delta_phi - j * L_sei * R_sei))
```
Comment (brosaplanella) [inline pybamm/models/submodels/interface/sei/sei_growth.py:92]: Speaking out of memory, should a similar factor be in the exponential to account for the temperature in the `R T / F`?

Comment (tinosulzer) [reply pybamm/models/submodels/interface/sei/sei_growth.py:92]: Oh yeah, it should only be inside the exponential right?

Comment (brosaplanella) [reply pybamm/models/submodels/interface/sei/sei_growth.py:92]: I am not sure about the outside as I don't remember the details of the model, but certainly inside the exponential. Are these from Scott's thesis?

Comment (tinosulzer) [reply pybamm/models/submodels/interface/sei/sei_growth.py:92]: The ones outside could be specified by making SEI reaction rate a function of temperature (e.g. arrhenius) I think?

Comment (brosaplanella) [reply pybamm/models/submodels/interface/sei/sei_growth.py:92]: Actually I don't know where do the ones outside come from exactly...

**Comment by tinosulzer** [comment]
will be fixed by #2104

**Pull request closed without merging**